.clippy.toml

```diff
@@ -1,4 +1,4 @@
 disallowed-methods = [
-  "futures::future::join_all",
-  "futures::future::try_join_all",
+  { path = "futures::future::join_all", reason = "We don't have a replacement for this method yet. Consider extending `SeqJoin` trait." },
+  { path = "futures::future::try_join_all", reason = "Use Context.try_join instead." },
 ]
```

Comment:
It might be worth a note here that there is a blanket waiver of `disallowed-methods` for tests, just in case somebody in the future intends to add a method to the list that they truly want to be disallowed everywhere.